<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-1TEHGD1" failures="1" tests="1" name="LoginTest" time="0.678" errors="0" timestamp="2024-11-29T20:21:21 IST">
  <testcase classname="org.test.comp.LoginTest" name="loginTest" time="0.0">
    <failure type="java.lang.RuntimeException" message="java.io.FileNotFoundException: D:\eclipse\BBEComSelE2E\src\test\resources\TestData\Productlist.xlsx (The process cannot access the file because it is being used by another process)">
      <![CDATA[java.lang.RuntimeException: java.io.FileNotFoundException: D:\eclipse\BBEComSelE2E\src\test\resources\TestData\Productlist.xlsx (The process cannot access the file because it is being used by another process)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.io.FileNotFoundException: D:\eclipse\BBEComSelE2E\src\test\resources\TestData\Productlist.xlsx (The process cannot access the file because it is being used by another process)
at java.base/java.io.FileOutputStream.open0(Native Method)
at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:184)
at org.apache.poi.openxml4j.opc.internal.FileHelper.copyFile(FileHelper.java:67)
at org.apache.poi.openxml4j.opc.ZipPackage.closeImpl(ZipPackage.java:435)
at org.apache.poi.openxml4j.opc.OPCPackage.close(OPCPackage.java:457)
at org.apache.poi.ooxml.POIXMLDocument.close(POIXMLDocument.java:204)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.close(XSSFWorkbook.java:594)
at org.utilities.com.JavaUtilities.readExcel(JavaUtilities.java:88)
at org.base.com.EngineClass.getData(EngineClass.java:51)
... 25 more
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- LoginTest -->
